src/impl/MilvusClientImpl.cpp

```diff
@@ -18,8 +18,15 @@
 
 namespace milvus {
 
-void
-MilvusClientImpl::Dummy() {
+std::shared_ptr<MilvusClient>
+MilvusClient::Create() {
+    return std::shared_ptr<MilvusClient>(new MilvusClientImpl());
```

Comment:
It's OK, but prefer to use std::make_shared instead of std::shared_ptr<>(new ). std::make_shared is exceptional safe.

Reply:
you are right